src/main/java/uk/gov/hmcts/reform/fees/client/FeesClientAutoConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.fees.client;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import feign.codec.Decoder;
+import feign.jackson.JacksonDecoder;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConditionalOnProperty(prefix = "fees", name = "api.url")
+@EnableFeignClients(basePackages = "uk.gov.hmcts.reform.fees.client")
+public class FeesClientAutoConfiguration {
+
+    @Bean
+    Decoder feignDecoder(ObjectMapper objectMapper) {
+        return new JacksonDecoder(objectMapper);
+    }
+
+    @Bean
+    public ObjectMapper objectMapper() {
+        return new ObjectMapper()
+                .registerModule(new JavaTimeModule())
+                .setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE)
```

Comment:
I am not sure whether we need this different configs for object mapper. As this is a library project and when used in any application i.e(claim-store), application context can have only one instance of this bean for auto wiring. which might have different needs. please verify